site/gatsby-site/jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
```

Comment:
What is consuming this config file? I don't see it referenced in this merge request.

Reply:
it helps VSCode to detect & provide alias imports.

Reply:
You point VSCode at this file, or it already knows about it?

Reply:
Please reference this [link](https://code.visualstudio.com/docs/languages/jsconfig#:~:text=The%20presence%20of%20jsconfig.,by%20the%20JavaScript%20language%20service.)